$NetBSD: patch-aa,v 1.3 2012/09/21 11:46:56 ryoon Exp $

Adapt ls(1) arguments for BSD.
Do not install man page gzipped.
Do not install into /usr.
Add case for NetBSD and DragonFly in the wrapper script.

--- INSTALL.orig	2012-03-28 10:48:06.000000000 +0000
+++ INSTALL
@@ -122,6 +122,8 @@ PrintRequiredFree()
 
         if [ "${OSname}" = "Linux" ] ; then
           size=`ls -lLn "$i" | ( read perm links owner group size date ; echo $size )`
+	elif [ `uname -s|sed -e 's|.*\(...\)|\\1|'` = "BSD" -o `uname -s` = "DragonFly" ] ; then
+	  size=`ls -lL "$i" | ( read perm links owner group size date ; echo $size )`
         else
           size=`ls -lLon "$i" | ( read perm links owner size date ; echo $size )`
         fi
@@ -637,10 +639,6 @@ InstallReader()
     "$ReadInstallDir" \
     "$ReadPlatformTar" "$SearchPlatformTar" "$CustomPlatformTar"
 
-  rm -f /usr/bin/acroread 2>/dev/null && \
-  ln -s "$ReadInstallDir/Adobe/Reader9/bin/acroread" /usr/bin/acroread 2>/dev/null && \
-  record_config remove /usr/bin/acroread
-
   if [ "${OSname}" = "Linux" ]; then
       rm -f /etc/bash_completion.d/acroread.sh 2>/dev/null && \
       ln -s "$ReadInstallDir/Adobe/Reader9/Resource/Shell/acroread_tab" /etc/bash_completion.d/acroread.sh 2>/dev/null && \
@@ -657,16 +655,8 @@ InstallReader()
           record_config remove /usr/local/etc/bash_completion.d/acroread.sh
       fi
   fi
-  if [ "${OSname}" = "Linux" ]; then
-    manPageFile="acroread.1.gz"
-  else
     manPageFile="acroread.1"
     gunzip "$ReadInstallDir/Adobe/Reader9/Resource/Shell/acroread.1.gz" 2>/dev/null 
-  fi
-
-  rm -f "/usr/share/man/man1/${manPageFile}" 2>/dev/null && \
-  ln -s "$ReadInstallDir/Adobe/Reader9/Resource/Shell/${manPageFile}" "/usr/share/man/man1/${manPageFile}" 2>/dev/null && \
-  record_config remove "/usr/share/man/man1/${manPageFile}"
 
   if [ "${OSname}" = "Linux" -o \( "${OSname}" = "SunOS" -a "`uname -m`" = "i86pc" \) ]; then
       printf "%s" "Setting up libraries ... "
@@ -950,6 +940,19 @@ case "$OSname" in
     ReadDefaultNum="$ReadIntelLinuxNum"
     DefaultName="Intel/Linux"
     ;;
+  NetBSD | DragonFly)
+    MachArch=`uname -p`
+    case "$MachArch" in
+     i386|x86_64)
+       ReadDefaultNum="$ReadIntelLinuxNum"
+       DefaultName="Intel/Linux"
+       ;;
+     sparc*)
+       ReadDefaultNum="$ReadSparcSolarisNum"
+       DefaultName="SPARC/Solaris 2.x"
+       ;;
+    esac
+    ;;
 esac
 
 if  [ -z "$ReadDefaultNum" ] ; then
